Flea/Pages/Home.razor

```diff
@@ -1,7 +1,62 @@
-@page "/"
-@page "/Home"
-<h1>Home</h1>
-
-@code {
-    
+@page "/"
+@page "/Home"
+
+<div>
+    <div class="view-head">
+        <input class="search-input" @bind-value="SearchTerm" @bind-value:event="oninput" placeholder="Søg efter loppemarked..."/>
+        <button class="button button--outline create-button" type="button" @onclick="AddEventItem">Tilføj nyt loppemarked</button>
+    </div>
+
+    <!-- Creates event items for each event that matches the user's search term -->
+    @if (DummyEventDB.Count == 0)
+    {
+        <h2>Der er ingen loppemarkeder at vise...</h2>
+    }
+    else if (FilteredEvents.Count == 0)
+    {
+        <h2>Der er ingen loppemarkeder, som matcher søgekriterierne...</h2>
+    }
+    else
+    {
+        foreach (DummyEvent e in FilteredEvents)
+        {
+            <div>
+                <EventListItem Date="@e.Date" RemainingTables="@e.RemainingTables" MissingPayments="@e.MissingPayments">
+                </EventListItem>
+            </div>
+        }
+    }
+</div>
+
+@code {
+    // Placeholder event class. Replace with Event.cs in Model
+    class DummyEvent
+    {
+        public DummyEvent(string date, int remainingTables, int missingPayments)
+        {
+            Date = date;
+            RemainingTables = remainingTables;
+            MissingPayments = missingPayments;
+        }
+        public string Date { get; }
+        public int RemainingTables { get; }
+        public int MissingPayments { get; }
+    }
+    
+    // Placeholder event database. 
+    private List<DummyEvent> DummyEventDB = new();
+
+    // SearchTerm is initialised as empty string to avoid null errors.
+    string SearchTerm { get; set; } = "";
+
+    // List of events from database that match the search term.
+    List<DummyEvent> FilteredEvents => 
+        DummyEventDB.Where(i => i.Date.ToLower().Contains(SearchTerm.ToLower())).ToList();
```

Comment:
Should all be marked explicitly with `private` and follow the proper naming conventions for private fields (`_camelCase`)